backend/server.js

```diff
@@ -87,6 +88,25 @@ app.get("/callback", async (req, res) => {
   }
 });
 
+app.get('/lyrics', async (req, res) => {
+  const { trackName, artistName } = req.query;
+
+  if (!trackName || !artistName) {
+    return res.status(400).json({ error: 'trackName and artistName are required' });
```

Comment:
It may be helpful to have a separate error for trackName and artistName if there would be any reason for them to diverge.

Reply:
okay